### PR TITLE
Mitz feature live chat

### PR DESCRIPTION
**Pull request by joemitz**
-added live chat feature with socketIO
-removed router test link from navbar
-added messages icon to navbar

### CODE REVIEW

**Review by rudyesar — APPROVED** [review]

File: tests/messages.test.js [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
-import Messages from '../client/src/messages/Messages.jsx';
-import ThreadList from '../client/src/messages/ThreadList.jsx';
-import ThreadTile from '../client/src/messages/ThreadTile.jsx';
+// import Messages from '../client/src/messages/Messages.jsx';
 import moment from 'moment';
 
-describe('Messages', ()=>{
+// describe('Messages', ()=>{
 
-  test('Should render all components with correct data', () => {
```
Comment (Chloe-Meinshausen) [inline tests/messages.test.js:14]: I created an Item especially for you in the sample data. ;)

Comment (joemitz) [reply tests/messages.test.js:14]: nice!

**Comment by Chloe-Meinshausen** [comment]
Hey Mitz, It's looking great.   My only comment is I do get CORS errors when I run the code locally.  - I vaguely remember CORS errors, but not enough to know if thats something on my end or yours. haha 

Access to XMLHttpRequest at 'http://127.0.0.1:3000/socket.io/?EIO=4&transport=polling&t=O4beDPZ' from origin 'http://localhost:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
127.0.0.1:3000/socket.io/?EIO=4&transport=polling&t=O4beDPZ:1          Failed to load resource: net::ERR_FAILED

**Comment by joemitz** [comment]
Thanks, I double checked and I had added a cors thing because I was having trouble at some point while implementing socketio, but it looks like that issue is gone so I got rid of cors stuff. Hopefully that fixes it

**Pull request merged**